src/dotnet-toolbox/UninstallCommand.cs

```diff
@@ -0,0 +1,98 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Runtime.InteropServices;
+using System.Threading.Tasks;
+using DotNetToolbox.Helpers;
+using Microsoft.Extensions.CommandLineUtils;
+using Newtonsoft.Json;
+
+namespace DotNetToolbox
+{
+    public class UninstallCommand : CommandLineApplication
+    {
+        private ToolboxConfiguration _toolboxConfig;
+        public UninstallCommand(CommandLineApplication parent)
+        {
+            Parent = parent;
+            Name = "uninstall";
+            PackageArgument = new PackageArgument(this);
+            Arguments.Add(PackageArgument);
```

Comment:
@danroth27 hmmm, one thing that comes to mind is what is the UX of this command if it needs to support both of these options?